### DISCUSSION TITLE
Tags Support

### DISCUSSION THREAD

**Discussion by xplosionmind** [maximevaillancourt/digital-garden-jekyll-template#82] [Ideas]
It would he very nice to integrate in the graph also tags, with a different color.

Tags should be discovered
- in the front matter, e.g. `tags: ["firsttag","secondtag"]` and accessed through the already existing liquid variable `{{ page.tags }}`
- automatically recognized inside the content of a post, when there's amy world preceded by a `#` which is not escaped. Then it should be automatically added to the `page.tags` array and the word inside the text transformed in an `<a>` tag, linking to `{{ site.base_url }}/tag/{{ tag-name }}`

**Comment by fabriceliut** [comment]
+1
